addons/api/addon/models/account.js

```diff
@@ -5,20 +5,43 @@
 
 import GeneratedAccountModel from '../generated/models/account';
 import { computed } from '@ember/object';
-import { equal } from '@ember/object/computed';
+import {
+  TYPE_AUTH_METHOD_PASSWORD,
+  TYPE_AUTH_METHOD_OIDC,
+  TYPE_AUTH_METHOD_LDAP,
+  TYPES_AUTH_METHOD,
+} from './auth-method';
 
 export default class AccountModel extends GeneratedAccountModel {
   // =attributes
 
   /**
    * @type {boolean}
    */
-  @equal('type', 'password') isPassword;
+  get isPassword() {
```

Comment:
would this not work? `@equal('type', TYPE_AUTH_METHOD_PASSWORD) isPassword`

Reply:
@DhariniJeeva Is there an effort to reduce computed ember magic?

Reply:
the existing computed property works, i refactored this to use a plain getter since computed properties are not necessary anymore to get a value.